_lifeofinternatpuppet/post_16.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: Week 16
+author: Disha-maker
+categories:
+  - team
+  - basics
+tags:
+  - week 16
+---
+
+Hello Everyone!
+
+Not too long to Christmas now, of course?
+
+This has been the busiest week of my placement so far, before the team wraps up for Christmas holidays!
+
+Some of the key highlights included:-
+
+- Module Release Prep
+    -[puppetlabs-motd](https://tickets.puppetlabs.com/browse/IAC-1308) - (v4.3.0)
+    -[puppetlabs-tomcat](https://tickets.puppetlabs.com/browse/IAC-1330)
+    (https://github.com/puppetlabs/puppetlabs-tomcat/pull/426) - (v4.3.0)
+    -[puppetlabs-tagmail](https://tickets.puppetlabs.com/browse/IAC-1349)
+    (https://github.com/puppetlabs/puppetlabs-tagmail/pull/196)
```

Comment:
There's some formatting issues with these bullet points. The rest will also need fixed up.
```suggestion
- [puppetlabs-motd](https://tickets.puppetlabs.com/browse/IAC-1308) - (v4.3.0)
- [puppetlabs-tomcat](https://tickets.puppetlabs.com/browse/IAC-1330)
    (https://github.com/puppetlabs/puppetlabs-tomcat/pull/426) - (v4.3.0)
- [puppetlabs-tagmail](https://tickets.puppetlabs.com/browse/IAC-1349)
    (https://github.com/puppetlabs/puppetlabs-tagmail/pull/196)
```

**[HINT!](https://github.com/puppetlabs/iac/pull/155#pullrequestreview-550000540)** 😉